data/geany-3.0.css

```diff
@@ -1,10 +0,0 @@
-@import "geany.css";
-
-/* make close button on the editor's tabs smaller */
-#geany-close-tab-button {
-	-GtkWidget-focus-padding: 0;
-	-GtkWidget-focus-line-width: 0;
-	-GtkButton-default-border: 0;
-	-GtkButton-default-outside-border: 0;
-	-GtkButton-inner-border: 0;
-}
```

Comment:
We provide custom names for all our windows just for that https://www.geany.org/manual/#defining-own-widget-styles-using-gtkrc-2-0

So you can do, for your suggested yellow button text in the prefs dialog:
```css
#GeanyDialogPrefs button {
  color: yellow;
}
```
So long as no other app name something "GeanyDialogPrefs", nothing else will be affected.  We have this for a long time just for this purpose -- style Geany and nothing else.